### ISSUE TITLE
MINOR: Publisher variable?

### ISSUE THREAD

**Issue by Kreyren** [PhoenicisOrg/scripts#806]
There are many games that are developed by 'one entity' and published by other.

Recommends making a publisher variable to differenciate editor from publisher + share more info about the game.

Reference https://github.com/PhoenicisOrg/scripts/pull/795/commits/ed16c2960a65cef415fbeb27732ffea5159330af#diff-c9ab1a1f58ebc7cd4cb8c6bded634778R21 Line 21

**Comment by Kreyren** [comment]
@plata I do xD Other then that probably not.. rather feature to be implemented once the phoenicis is in gold i would say :thinking:

**Comment by plata** [comment]
What do you want to do with it? I mean if you look at an application in Phoenicis: how does it help you to know who published it?

**Comment by Kreyren** [comment]
@plata Yes since it may help in identifying the dependencies, game quality, etc. 

... It will probably look in GUI good too

but it's still minor it's up to you

**Comment by plata** [comment]
I think we can do it but then someone has to update all the apps...

**Comment by Kreyren** [comment]
@plata Well i can help it seems to be effective to add it.

**Comment by plata** [comment]
@qparis @ImperatorS79 @madoar shall we do this or not?

**Comment by qparis** [comment]
Don’t think that it is relevant for the moment

**Comment by ImperatorS79** [comment]
No opinion.

**Comment by madoar** [comment]
What do we lose by adding a `publisher` field?

**Comment by plata** [comment]
@madoar the question is more: what would we gain? We can also add release date of the application and a ton of other information but is it useful for anything?

**Comment by Kreyren** [comment]
@plata 
> We can also add release date of the application and a ton of other information but is it useful for anything?

Depending on the info provided yes it's very usefull for lots of usecases like modding, wine/script development, presentation, etc..

I woudn't include it now since it's minor and can be included with low priority. -> should be included once phoenicis's core functions are stable and usable.

**Comment by madoar** [comment]
I think it is useful. Sometimes there are multiple applications with the same name. Maybe even scripts.
In such cases it is useful to have such additional information.

As far as I understand the corresponding code it should only be a matter of adding an additional field to the JS prototypes and maybe also to the Java interfaces.

**Comment by qparis** [comment]
Agree with @plata. We have a lot of priority we need to focus on now and the balance value added / developement time is not interesting enough comparing the other subjects we have. If someone wants to do it, they are free to do so. In the mean time I suggest we decide that it is not our core priority

**Comment by plata** [comment]
So we keep the issue?
